File: src/main/java/dev/slne/discord/exception/ticket/DeleteTicketChannelException.java
package dev.slne.discord.exception.ticket;

import dev.slne.discord.exception.DiscordException;
import java.io.Serial;

public class DeleteTicketChannelException extends DiscordException {

  @Serial
  private static final long serialVersionUID = -2980471014795318817L;

  public DeleteTicketChannelException(String message) {
    super(message);
  }

  public DeleteTicketChannelException(String message, Throwable cause) {
    super(message, cause);
  }

  public DeleteTicketChannelException(DiscordException exception) {
    super(exception);
  }
}

File: src/main/kotlin/dev/slne/discord/exception/ticket/DeleteTicketChannelException.kt
package dev.slne.discord.exception.ticket

import dev.slne.discord.exception.DiscordException
import java.io.Serial

class DeleteTicketChannelException : DiscordException {
    constructor(message: String?) : super(message)

    constructor(message: String?, cause: Throwable?) : super(message, cause)

    constructor(exception: DiscordException) : super(exception)

    companion object {
        @Serial
        private val serialVersionUID = -2980471014795318817L
    }
}
